jetson/percep/test_protocol.md

```diff
@@ -0,0 +1,73 @@
+## Compile Test
+- [ ] ` ./jarvis build jetson/percep -o with_zed=false ar_detection=true obs_detection=true perception_debug=true `
+  - [ ] No compiler errors or warnings
```

Comment:
Probably want to add a little more - since in this line `perception_debug=true` but in the next line it's false

Reply:
Got it